src/App.tsx

```diff
@@ -2,7 +2,7 @@ import { ResourceSchema } from "./state/openapi";
 import SpecSpecifierPage from "./app/spec_specifier/page";
 import { createBrowserRouter, RouteObject, RouterProvider } from "react-router-dom";
 import Layout from "./app/explorer/page";
-import ResourceList from "./app/explorer/resource_list";
```

Comment:
should the file be renamed too? I guess this is something you'd expect more with a directory structure (`app/explorer/page/resource_list`), but I don't know a lot about vite best practices.

Reply:
We should probably have a better file structure. I'll keep this as-is just to get this merged